lms/djangoapps/courseware/date_summary.py

```diff
@@ -110,6 +120,7 @@ class TodaysDate(DateSummary):
     """
     css_class = 'todays-date'
     is_enabled = True
+    date_format = '%b %d, %Y (%H:%M UTC)'
```

Comment:
I don't think so. Given that we (English speakers) use the French acronym, I'd assume others do too. I'll do some light research, though.